database/src/main/resources/db/migration/V1_82__gtfs_package_tables.sql

```diff
@@ -0,0 +1,141 @@
+-- Update gtfs_package to work with foreign keys
+
+ALTER TABLE "gtfs_package"
```

Comment:
combine 81 and 82 migrations, no need to alter gtfs_package just change the definition (it is not yet in master)